test/json_patch_test.dart

```diff
@@ -1,10 +1,10 @@
 import 'dart:convert';
 
-import 'package:test/test.dart';
 import 'package:json_patch/json_patch.dart';
+import 'package:test/test.dart';
 
 void main() {
-  group('JsonPatch', () {
+  group('.diff', () {
```

Comment:
This `'JsonPatch'` was deliberate, as it is the class being tested, and each test had `'.diff'` to describe which method is being tested for what.

Reply:
Ah cool! I'll change that back 😄